source/includes/common/bucket-replication.rst

```diff
@@ -0,0 +1,181 @@
+.. start-create-replication-remote-targets-cli-desc
+
+Use the :mc-cmd:`mc admin bucket remote add` command to create a replication target from each deployment to the appropriate bucket on the destination deployment. 
+A bucket may have multiple remote targets to different target buckets.
+No two targets can resolve from one bucket to the same remote bucket. 
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin bucket remote add ALIAS/BUCKET                    \
+      https://RemoteUser:Password@HOSTNAME/BUCKETDESTINATION  \
+      --service "replication"
+
+- Replace ``ALIAS`` with the :ref:`alias <alias>` of the MinIO deployment that acts as the origin for the replication.
+- Replace ``BUCKET`` with the name of the bucket to replicate from on the origin deployment.
+- Replacete ``RemoteUser`` with the user name that has the :ref:`necessary replication permissions <minio-bucket-replication-serverside-twoway-permissions>`
+- Replace ``Password`` with the secret key for the ``RemoteUser``.
+- Replace ``HOSTNAME`` with the URL of the destination deployment.
+- Replace ``BUCKETDESTINATION`` with the name of the bucket to replicate to on the destination deployment.
+
+The command returns an :abbr:`ARN <Amazon Resource Name>` similar to the following:
+
+.. code-block:: shell
+
+   Role ARN = 'arn:minio:replication::<UUID>:BUCKET'
+
+Copy the ARN to use in the next step, noting the MinIO deployment.
+
+.. end-create-replication-remote-targets-cli-desc
+
+.. start-create-bucket-replication-rule-console-desc
+
+#. Log in to the MinIO Console for the deployment
```

Comment:
Secondary thought - replace these with a b c blocks. The numbering looks a little odd otherwise when rendered.

e.g.

a.
b.
c.

So for the header, you have 1) . a, 1).b, 1).c , etc.